app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsTable/LocalUnitTableActions/index.tsx

```diff
@@ -76,6 +86,13 @@ function LocalUnitsTableActions(props: Props) {
         [setShowLocalUnitModalTrue],
     );
 
+    const handleDeleteLocalUnitClick = useCallback(
+        () => {
+            setShowDeleteLocalUnitModalTrue();
+        },
+        [setShowDeleteLocalUnitModalTrue],
+    );
```

Comment:
We don't need to define this function. We can directly pass `setShowDeleteLocalUnitModalTrue` where `hanldeDeleteLocalUnitClick` is used